examples/00-basic-examples/HFSS_CoordinateSystem.py

```diff
@@ -19,134 +18,150 @@
 d = Desktop("2021.1", NG=nongraphical)
 
-#####################################################
-# Insert HFSS design
-# ------------------
-# This command insert an hfss design with a default name.
+###############################################################################
+# Insert an HFSS design.
+# ----------------------
+# This command inserts an HFSS design with the default name.
 hfss = Hfss()
 
-#####################################################
-# Create a Coordinate System
-# --------------------------
-# The coordinate system is centered on Global origin and has the axis aligned to Global
-# the coordinate system is saved in the object cs1
+###############################################################################
+# Create a coordinate system.
+# ---------------------------
+# The coordinate system is centered on the ``Global`` origin and has the axis 
+# aligned to the ``Global`` coordinate system.
+# The new coordinate system is saved in the object `cs1`.
+
 cs1 = hfss.modeler.create_coordinate_system()
 
-#####################################################
-# Edit the Coordinate System
-# --------------------------
-# The cs1 object expose properties and methods to manipulate the coordinate system.
-# The origin can be changed
+###############################################################################
+# Modify the coordinate system.
+# -----------------------------
+# The `cs1` object exposes properties and methods to manipulate the coordinate system.
+# The origin can be changed.
+
 cs1.props["OriginX"] = 10
 cs1.props["OriginY"] = 10
 cs1.props["OriginZ"] = 10
 cs1.update()
 
-# The pointing vectors can be changed
+# The pointing vectors can be changed.
+
 ypoint = [0, -1, 0]
 cs1.props["YAxisXvec"] = ypoint[0]
 cs1.props["YAxisYvec"] = ypoint[1]
 cs1.props["YAxisZvec"] = ypoint[2]
 cs1.update()
 
-#####################################################
-# Rename the Coordinate System mode
-# ---------------------------------
-# The name can be changed
+###############################################################################
+# Rename the coordinate system.
+# -----------------------------
+
 cs1.rename('newCS')
 
-#####################################################
-# Change the Coordinate System mode
-# ---------------------------------
-# Use the option to select the mode: 0 for "Axis/Position", 1 for "Euler Angle ZXZ", 2 for "Euler Angle ZYZ"
-# This will change to "Euler Angle ZXZ"
+###############################################################################
+# Change the coordinate system mode.
+# ----------------------------------
+# Use the function `change_cs_mode` to change the mode: ``0`` for "Axis/Position", 
+# ``1`` for "Euler Angle ZXZ", or ``2`` for "Euler Angle ZYZ".
+# This command changes the mode to "Euler Angle ZXZ".
+
 cs1.change_cs_mode(1)
 
-# With the new mode there are new properties that can be edited.
+# In the new mode, there are properties that can be edited.
+
```

Comment:
We should not add whitespace after comments like this.  These comments will be rendered in "code" styling rather than plain text and are more "developer" notes.